lib/views/forget_password/forget_password_view.dart

```diff
@@ -0,0 +1,146 @@
+import 'dart:ui';
+import 'package:flutter/material.dart';
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+import 'package:kayla_flutter_ic/gen/assets.gen.dart';
+import 'package:kayla_flutter_ic/views/common/build_context_ext.dart';
+import 'package:kayla_flutter_ic/views/forget_password/forget_password_form.dart';
+import 'package:kayla_flutter_ic/views/forget_password/forget_password_state.dart';
+import 'package:kayla_flutter_ic/views/forget_password/forget_password_view_model.dart';
+
+final forgetPasswordViewModelProvider = StateNotifierProvider.autoDispose<
+    ForgetPasswordViewModel, ForgetPasswordState>(
+  (_) => ForgetPasswordViewModel(),
+);
+
+class ForgetPasswordView extends ConsumerStatefulWidget {
+  const ForgetPasswordView({super.key});
+
+  @override
+  ForgetPasswordViewState createState() => ForgetPasswordViewState();
+}
+
+class ForgetPasswordViewState extends ConsumerState<ForgetPasswordView>
+    with TickerProviderStateMixin {
+  AppBar get _appBar => AppBar(
+        leading: const BackButton(),
+        backgroundColor: Colors.transparent,
+        shadowColor: Colors.transparent,
+      );
+
+  Widget get _background => SizedBox(
+        width: MediaQuery.of(context).size.width,
+        height: MediaQuery.of(context).size.height,
+        child: Stack(
+          children: [
+            _backgroundImage,
+            _linearGradientBlurBackground,
+          ],
+        ),
+      );
+
+  Widget get _backgroundImage => Image(
+        image: Assets.images.nimbleBackground.image().image,
+        fit: BoxFit.cover,
+        alignment: Alignment.center,
+      );
+
+  Widget get _linearGradientBlurBackground => Stack(
```

Comment:
@nmint8m I think we can define this as a common widget for reusable (login/forget password using the same?) 🤔